.github/workflows/nightly-test.yaml

```diff
@@ -14,58 +19,86 @@ jobs:
       matrix:
         os: ["ubuntu:20.04", "ubuntu:22.04", "ubuntu:24.04"]
         arch: ["amd64", "arm64"]
-        release: ["latest/edge"]
+        channel: ["latest/edge"]
       fail-fast: false # TODO: remove once arm64 works
+    uses: ./.github/workflows/e2e-tests.yaml
+    with:
+      arch: ${{ matrix.arch }}
+      os: ${{ matrix.os }}
+      channel: ${{ matrix.channel }}
+      test-tags: 'up_to_nightly'
 
-    runs-on: ${{ matrix.arch == 'arm64' && 'self-hosted-linux-arm64-jammy-large' || 'self-hosted-linux-amd64-jammy-large' }}
+  Trivy:
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+    strategy:
+      matrix:
+        include:
+          # Latest branches
+          - { branch: main, channel: latest/edge }
+          # Stable branches
+          # Add branches to test here
+          - { branch: release-1.30, channel: 1.30-classic/edge }
+          - { branch: release-1.31, channel: 1.31-classic/edge }
```

Comment:
Are we missing 1.32 here? @bschimke95

Reply:
Yes

Reply:
Missed that, thanks! 1.32 wasn't released when I prepared the first commit.

Reply:
yes, this eventually needs to be automated. 
@petrutlucian94 let's add a todo for follow up work. This PR is already large enough.

Reply:
I think it makes sense to have a separate "get-latest-releases" action that can be reused across the workflows as we likely need this in other places as well.

Reply:
Let's use the automation in Microk8s workflows as well. We have the same issues there.